cli/ballerina-packerina/src/main/java/org/ballerinalang/packerina/task/RunExecutableTask.java

```diff
@@ -130,6 +130,10 @@ private void runGeneratedExecutable(BLangPackage executableModule, BuildContext
             ProcessBuilder pb = new ProcessBuilder(commands).inheritIO();
             Process process = pb.start();
             process.waitFor();
+            int exitValue = process.exitValue();
+            if (exitValue != 0) {
+                Runtime.getRuntime().exit(exitValue);
+            }
         } catch (IOException | InterruptedException e) {
```

Comment:
Don't we need to exit if get exception while running the command?

Reply:
The thrown exceptions are handled and exits are covered at the main class at https://github.com/ballerina-platform/ballerina-lang/blob/ballerina-1.2.x/cli/ballerina-tool/src/main/java/org/ballerinalang/tool/Main.java

But when there is no exception thrown as in the case when the ballerina program panics with an error, the exit code does not reflect the actual status. That's why I am suggesting this fix.

Reply:
I realised, we can throw an exception when exit code is not zero and let the main class handle it to keep it consistent. I'll fix it.